.dockerignore

```diff
@@ -0,0 +1,4 @@
+.git/*
+.pg_data/*
+log/*
+tmp/*
```

Comment:
We should probably also ignore `node_modules` now that we're running Yarn in the container.